src/backend/model_deployments/cohere_platform.py

```diff
@@ -90,11 +95,11 @@ async def invoke_chat_stream(
             )
             yield to_dict(event)
 
+    @collect_metrics_rerank
```

Comment:
We need the decorator in all model deployments for consistency

Reply:
addressed